README.md

```diff
@@ -192,7 +192,7 @@ $SERVICE_BINDING_ROOT
 
 A Service Binding describes the connection between a [Provisioned Service](#provisioned-service) and an [Application Projection](#application-projection).  It **MUST** be codified as a concrete resource type with API version `service.binding/v1alpha2` and kind `ServiceBinding`.  Multiple Service Bindings can refer to the same service.  Multiple Service Bindings can refer to the same application.  An exemplar CRD can be found [here][sb-crd].
 
-Restricting service binding to resources within the same namespace is strongly **RECOMMENDED**.  Cross-namespace service binding **SHOULD** be secured appropriately by the implementor to prevent attacks like privilege escalation and secret enumeration.
+Restricting service binding to resources within the same namespace is strongly **RECOMMENDED**.  However, if implemented, cross-namespace service binding **SHOULD** be secured appropriately by the implementor to prevent attacks like privilege escalation and secret enumeration.
```

Comment:
I think security is a major concern, but there's also a usability concern in terms of looking at a `ServiceBinding` CR and being able to determine which service that maps to - i.e. not a magically / hidden lookup of services in other namespaces.

```suggestion
Restricting service binding to resources within the same namespace is strongly **RECOMMENDED**.  Implementations that choose to support cross-namespace service binding **SHOULD** provide a security model that prevents attacks like privilege escalation and secret enumeration, as well as a deterministic way to declare target namespaces.
```

Reply:
The change looks good to me. I have committed it. Thanks @arthurdm !